core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/encrypter/EncryptionController.java

```diff
@@ -0,0 +1,35 @@
+package io.metadew.iesi.server.rest.encrypter;
+
+import io.metadew.iesi.server.rest.encrypter.dto.EncryptDto;
+import io.metadew.iesi.server.rest.encrypter.dto.EncryptPostDto;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+/**
+ * The encryption controller is used to provide encrypted text.
+ * @author Suyash.d.jain
+ */
+
+@RestController
+@Tag(name = "encrypt", description = "controller to encrypt text")
+@RequestMapping("/encrypt")
+@ConditionalOnWebApplication
+public class EncryptionController {
+
+    private final IEncryptionService encryptionService;
+
+    public EncryptionController(IEncryptionService encryptionService) {
+        this.encryptionService = encryptionService;
+    }
+
+    @PostMapping("")
+    public EncryptDto getEncryptedPassword(@Valid @RequestBody EncryptPostDto encryptPostDto){
```

Comment:
Please call the method: encryptPassword. We don't really get the encrypted password, we encrypt the password and want the api to return that encrypted password

Reply:
We are passing some input and on the basic of that we are getting encrypted password. So name will remain getEncryptedPassword.

Reply:
Yes but the action is to encrypt password not to get the encrypted password. When we create a script, he also have a scriptDto that has returned, and we don't call this method getCreatedScript.